### DISCUSSION TITLE
Watchtower not updating container with newest image from private registry?

### DISCUSSION THREAD

**Discussion by calicoder77** [containrrr/watchtower#1349] [Q&A] [answered]
Good day folks, I just recently set up watchtower for a container with an image file I pulled from a private container registry (self-hosted Gitlab) with: `docker run -d --name myProject -e REPO_USER=******** -e REPO_PASS=******** -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower hostname:5050/myProject`

I'm able to login, pull and push to that `hostname:5050/myProject` container registry. Thing is any new images added to that container registry doesn't trigger watchtower to pull the new image and remake the container for me. What am I missing here? Can Watchtower sense networked container registry additions?

Thanks again for the help, incredible project btw!

**Comment by piksel** [comment, marked as answer]
The non-flag arguments to watchtower is the names of containers to watch, you are passing in `hostname:5050/myProject` which I assume is the image URL on the repository? Replace it with your containers name, or omit it to update all containers.
